<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateContactInfo</class>
 <widget class="QDialog" name="updateContactInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>261</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Update contact</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>190</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="firstName">
       <property name="placeholderText">
        <string>First Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lastName">
       <property name="placeholderText">
        <string>Last Name</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="phone">
         <property name="placeholderText">
          <string>Phone Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Provider</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Verizon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T-Mobile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Metro PCS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sprint</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="email">
       <property name="placeholderText">
        <string>Email</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>firstName</tabstop>
  <tabstop>lastName</tabstop>
  <tabstop>phone</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>email</tabstop>
  <tabstop>updateButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
